src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -320,10 +321,15 @@ You may have to run ${chalk.greenBright(
       this.currentRequestPromise.cancel()
     }
   }
-
-  private async resolvePrismaPath(): Promise<string> {
+  // TODO Clean Up Logic
```

Comment:
Do you still want to clean that up before merging?

Reply:
Will remove the comment